### ISSUE TITLE
Disable SSL certificate verification

### ISSUE THREAD

**Issue by mattheo-geoffray** [blackfireio/player#40]
Is that possible to disable SSL certificate verification, like `-k` option with `cURL`?   
   
```bash
blackfire-player run scenario.bkf --endpoint=https://webserver/
Blackfire Player

Scenario  "Authentication"
  Failure on step  "Login page"  defined in scenario.bkf at line  7
  └ cURL error 60: SSL certificate problem: self signed certificate (see http://curl.haxx.se/libcurl/c/libcurl-errors.html)

 KO  Scenarios  1  - Steps  1  - Failures  1
```

**Comment by romainneutron** [comment]
Fixed as of v1.3.0
https://github.com/blackfireio/player/releases/tag/v1.3.0

**Comment by mattheo-geoffray** [comment]
@romainneutron @iamluc Thank you for the very very quick release! !
